<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StorageConfigWidget</class>
 <widget class="QWidget" name="StorageConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>769</width>
    <height>561</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="2,2,1,0,0">
   <item>
    <widget class="QGroupBox" name="storagesGroupBox">
     <property name="title">
      <string>Main Storages</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0,0">
      <item>
       <widget class="QnTableView" name="mainStoragesTable">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::NoSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>100</number>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="addExtStorageToMainBtn">
          <property name="text">
           <string>Add External Storage</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="rebuildMainButton">
          <property name="text">
           <string>Rebuild Archive Index</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QnStorageRebuildWidget" name="rebuildMainWidget" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="backupStoragesGroupBox">
     <property name="title">
      <string>Backup Storages</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,0,0">
      <item>
       <widget class="QnTableView" name="backupStoragesTable">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::NoSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="rowCount" stdset="0">
         <number>0</number>
        </property>
        <property name="columnCount" stdset="0">
         <number>7</number>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>100</number>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="addExtStorageToBackupBtn">
          <property name="text">
           <string>Add External Storage</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="rebuildBackupButton">
          <property name="text">
           <string>Rebuild Archive Index</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QnStorageRebuildWidget" name="rebuildBackupWidget" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="backupControls" native="true">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QLabel" name="backupTimeLabel"/>
      </item>
      <item>
       <widget class="QStackedWidget" name="stackedWidgetBackupInfo">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="backupPreparePage">
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>1000</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="backupWarningLabel">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="backupProgressPage">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Backup progress</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QProgressBar" name="progressBarBackup">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="value">
               <number>0</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="backupStopButton">
              <property name="text">
               <string>Cancel</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QPushButton" name="camerasToBackupButton">
          <property name="text">
           <string notr="true">&lt;Cameras to Backup...&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="backupHSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="whenToBackupLabel">
          <property name="text">
           <string>Perform backup</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxBackupType"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonSchedule">
          <property name="text">
           <string>Schedule...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="backupStartButton">
          <property name="text">
           <string>Start Backup Now</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="backupOptionLabel">
     <property name="text">
      <string>Storage backup is available if there are 2 or more storages in the system.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="storagesWarningLabel">
     <property name="text">
      <string>Unchecking the storage will cause nothing but writing stop.
It won't deny oldest footage removing from the drive.</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QnTableView</class>
   <extends>QTableView</extends>
   <header location="global">ui/widgets/table_view.h</header>
  </customwidget>
  <customwidget>
   <class>QnStorageRebuildWidget</class>
   <extends>QWidget</extends>
   <header location="global">ui/widgets/properties/storage_rebuild_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
